src/components/PromptForm/PromptSendButton.tsx

```diff
@@ -308,89 +283,16 @@ function DesktopPromptSendButton({ isLoading }: PromptSendButtonProps) {
-        <MenuList
-          maxHeight={"80vh"}
-          overflowY={"auto"}
-          zIndex={theme.zIndices.dropdown}
-          onKeyDownCapture={onStartTyping}
-        >
-          <MenuGroup title="Providers">
-            {Object.entries(providersList).map(([providerName, providerObject]) => (
-              <MenuItem
-                paddingInline={4}
-                key={providerName}
-                onClick={() => {
-                  setSettings({ ...settings, currentProvider: providerObject });
-                }}
-              >
-                {settings.currentProvider.name === providerName ? (
-                  <IoMdCheckmark style={{ marginRight: "0.6rem" }} />
-                ) : (
-                  <span style={{ width: "1.6rem", display: "inline-block" }} />
-                )}
-                {providerName}
-              </MenuItem>
-            ))}
-          </MenuGroup>
-          <MenuDivider />
-          <MenuGroup title="Models">
-            <InputGroup>
-              <InputLeftElement paddingLeft={3} pointerEvents="none">
-                <TbSearch />
-              </InputLeftElement>
-              <Input
-                marginInline={2}
-                marginBottom={1}
-                ref={inputRef}
-                type="text"
-                variant="outline"
-                placeholder="Search models..."
-                value={searchQuery}
-                onChange={(e) => {
-                  e.preventDefault();
-                  setSearchQuery(e.target.value);
-                }}
-              />
-            </InputGroup>
-            <Box maxHeight="40vh" overflowY="auto">
-              {models
-                .filter((model) => isChatModel(model.id))
-                .filter((model) =>
-                  model.name.toLowerCase().includes(debouncedSearchQuery.toLowerCase())
-                )
-                .map((model) => (
-                  <MenuItem
-                    paddingInline={4}
-                    closeOnSelect={true}
-                    key={model.id}
-                    onClick={() => setSettings({ ...settings, model })}
-                  >
-                    {settings.model.id === model.id ? (
-                      <IoMdCheckmark style={{ marginRight: "0.6rem" }} />
-                    ) : (
-                      <span
-                        style={{
-                          paddingLeft: "1.6rem",
-                          display: "inline-block",
-                        }}
-                      />
-                    )}
-                    {model.name}
-                  </MenuItem>
-                ))}
-            </Box>
-          </MenuGroup>
-        </MenuList>
-      </Menu>
+      {
+        <Box>
+          <ModelProviderMenu
+            onItemSelect={(modelId) => {
+              const model = models.find((m) => m.id === modelId);
+              if (model) setSettings({ ...settings, model });
+            }}
+          />
+        </Box>
```

Comment:
```suggestion
          <ModelProviderMenu
            onItemSelect={(modelId) => {
              const model = models.find((m) => m.id === modelId);
              if (model) setSettings({ ...settings, model });
            }}
          />
```

Get rid of this `Box` to solve border issues.

Reply:
Understood.

Reply:
This is fixed in commit https://github.com/tarasglek/chatcraft.org/pull/730/commits/5a286eb40b8abf796c9e9558a1a20320668ffc0e